### DISCUSSION TITLE
SQL Join with condition

### DISCUSSION THREAD

**Discussion by corven13** [goravel/goravel#333] [Q&A] [answered]
I have a problem querying the data. I tried to query on pgAdmin is fine but I can't add a condition when using Join on Goravel.

Example on pgAdmin:
`LEFT JOIN daily_reports ON daily_reports.field_group_id = field_groups.id AND daily_reports.date = '2023-12-03'`

Example on Goravel:
`Join("LEFT JOIN daily_reports ON daily_reports.field_group_id = field_groups.id AND daily_reports.date = ?", date).`

I would like to add a condition after the join command.
Does anyone have an idea or solution for this issue?

**Comment by hwbrzzl** [comment]
Please show your complete code and the specific error.

**Reply by corven13** [reply, marked as answer]
Issue solved! I forgot to add another condition on the Join command. That is why the results are incorrect.
This solution works on ORM.

`LEFT JOIN daily_reports ON daily_reports.field_group_id = field_groups.id AND daily_reports.date = '2023-12-03'`
